### PR TITLE
Initial implementation of checkout.com

### PR DESCRIPTION
**Pull request by andrzejewsky**

### CODE REVIEW

File: packages/commercetools/theme/nuxt.config.js [hunk 1]
```diff
@@ -58,12 +63,14 @@ export default {
...
   ],
   modules: [
+    ['@vue-storefront/checkout-com/nuxt', { publicKey: 'pk_test_8638c4e2-e44a-407f-a5f3-594a8503bcd0' }],
     'nuxt-i18n',
     'cookie-universal-nuxt',
     'vue-scrollto/nuxt'
   ],
   build: {
     transpile: [
+      // '@vue-storefront/checkout-com-nuxt',
```
Comment (filrak) [inline packages/commercetools/theme/nuxt.config.js:73]:
```suggestion
```

File: packages/commercetools/theme/package.json [hunk 2]
```diff
@@ -15,6 +15,8 @@
   "dependencies": {
     "@storefront-ui/vue": "^0.7.10",
     "@vue-storefront/commercetools": "^0.0.3",
+    "@vue-storefront/checkout-com": "^0.0.3",
+    "@vue-storefront/checkout-com-nuxt": "^0.0.3",
```
Comment (filrak) [inline packages/commercetools/theme/package.json:19]:
```suggestion
```

File: packages/commercetools/theme/nuxt.config.js (hunk 1 above)
Comment (Fifciu) [reply packages/commercetools/theme/nuxt.config.js:73]: Just commited

File: packages/commercetools/theme/package.json (hunk 2 above)
Comment (Fifciu) [reply packages/commercetools/theme/package.json:19]: Just commited

**Comment by Fifciu** [comment]
## Saving cards implementation plan
- [ ] `save_payment_instrument` should be false for Guest and should depend on some checkbox for Customer
- [ ] Create serverMiddleware that changes token to the customer_id, then send request to https://docs.checkout.com/preview/commercetools#Commercetools-Retrievesavedpaymentinstruments from the server and respond with it
- [ ] Create serverMiddleware that changes token to the customer_id, then send request to https://docs.checkout.com/preview/commercetools#Commercetools-Deleteasavedpaymentinstrument from the server and respond with it
- [ ] Prepare functions to send request to these endpoints 
- [ ] Modify `createPayment` to support saved credit card

**Pull request merged**